doc/howtos/booting-and-installation.rst

```diff
@@ -141,7 +141,42 @@ Flashing an Intel Edison requires use of a breakout board and two micro-USB cabl
 #. Plug in the second micro-USB cable to the J16 connector as instructed by the running flashall script
 #. Wait for all the images to flash. You will see the progress on both the flasher and on the serial console
 #. Once flashing is done, the image will automatically boot up and auto-login as ``root``, no password is required
-     
+
+BeagleBone
+==========
+
+BeagleBone is booted from a microSD card. Partitions are MSDOS, not GPT.
+
+#. Create two partitions, 64 MB primary partition of type 6 (FAT16) and make it bootable, rest of the space as primary partition of type 83 (Linux)
```

Comment:
I'd suggest we leave the exercise of adding instructions for Windows and MAC users to a later point in time.